swerve_controller/CMakeLists.txt

```diff
@@ -0,0 +1,64 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(swerve_controller)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+# Package dependencies
+set(${PROJECT_NAME}_CATKIN_DEPS
+    controller_interface
+    joint_state_controller
+    nav_msgs
+    realtime_tools
+    robot_state_publisher
+    roslint
+    realtime_tools
+    tf
+    urdf_geometry_parser)
+list(SORT ${PROJECT_NAME}_CATKIN_DEPS)
+find_package(catkin REQUIRED COMPONENTS ${${PROJECT_NAME}_CATKIN_DEPS})
+
+# Create the package
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  CATKIN_DEPENDS ${${PROJECT_NAME}_CATKIN_DEPS}
+)
+
+# Source and headers for build
+include_directories(include)
+include_directories(${catkin_INCLUDE_DIRS})
```

Comment:
only one `include_directories` should be there

Reply:
I will merge it